Amplify/Categories/DataStore/Model/Model.swift

```diff
@@ -18,6 +18,10 @@ public protocol Model: Codable {
     /// A reference to the `ModelSchema` associated with this model.
     static var schema: ModelSchema { get }
 
+    /// The reference to the root path. It might be `nil` if models do not support
+    /// property references.
+    static var rootPath: PropertyContainerPath? { get }
+
```

Comment:
This is a breaking change. Is there a supported use case for customer adding `Model` conformance to their own types? If not, is there a reason `Model` doesn't have the same disclaimer that other "public but not actually public" types have?

Reply:
I missed the default implementation - disregard.